marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.tsx

```diff
@@ -345,6 +345,7 @@ const MappingCard: React.FC<Props> = (props) => {
 
     // construct infinitely nested source Data
     const generateNestedDataSource = (respData, nestedDoc: Array<any>, parentNamespace = namespaceString) => {
+        console.log("DATA: ", respData)
```

Comment:
I think these are typically removed in the UI code too before merging to develop?

Reply:
Oops, accidentally left that in; thanks for catching it!